components/monitoring/README.md

```diff
@@ -40,160 +42,138 @@ for adding the service monitor declaration:
 
-  - Add tiles on the dashboard to track your initial set of service level indicators. If you correctly added your servicemonitor to the stage Prometheus datasource, it will show up in the Query list when you edit a tile.
+### 4. Grafana dashboards
 
-  - Export the dashboard definition in JSON format. (At the top of the screen, the icon with 3 dots lets you "Share dashboard or panel". Select Export... Save to file.)
+A dashboard is a set of one or more panels organized and arranged into one or more rows. 
+It visualizes results from multiple data sources simultaneously.
+New dashboards can be added through the user interface, preconfigured in infra-deployment project, 
+or imported from other projects.
 
-  -  Store the dashboard definition in git, in infra-deployments. 
 
-  - Add your dashboard to the [kustomization file](https://github.com/redhat-appstudio/infra-deployments/blob/main/components/monitoring/grafana/base/kustomization.yaml#L15) to automatically add it to future deployments.
+#### Create a dashboard
 
-  - [Here](https://github.com/redhat-appstudio/infra-deployments/blob/main/components/monitoring/grafana/base/dashboards/example.json) is an example for the default dashboard.
+  - [Create a dashboard](https://grafana.com/docs/grafana/v9.0/dashboards/)
+  for your team's view of your service's Service Level Indicators.
+  (After navigating to your folder, + Create Dashboard)  
+  ***Note:***  
+  Creating a new dashboard is available only for development environment.  
+  You may consider editing the existing `example` dashboard for staging and production environments  
+  The `example` dashboard json deifinition can be found 
+  [here](https://github.com/redhat-appstudio/infra-deployments/blob/main/components/monitoring/grafana/base/dashboards/example.json)
 
-##### In place configuration
+  - Add tiles to the dashboard to track your initial set of service level indicators.  
+  If the servicemonitor was added ocrrectly to the stage Prometheus datasource, 
+  it will show up in the query list when you edit a tile.
 
-  - Add dashboard json to the [dashboards](https://github.com/redhat-appstudio/infra-deployments/tree/main/components/monitoring/grafana/base/dashboards) folder
+  - Export the dashboard definition in JSON format. 
+  (At the top of the screen, the icon with 3 dots lets you "Share dashboard or panel". Select Export... Save to file.)
+   
 
-  - Add dashboard's file name to [kustomization.yaml](https://github.com/redhat-appstudio/infra-deployments/blob/main/components/monitoring/grafana/base/kustomization.yaml#L15)
-    ```yaml
-    configMapGenerator:
-    - name: grafana-dashboard-definitions
-      files:
-      - example.json=dashboards/example.json
+#### Team's repository
+
+Follow the next steps to define a dashboard in your team's repository
+
+  - The dashboard should be located in the team’s repository, no change in `infra-deployments` is required,
```

Comment:
As those steps should be done one-after-the-other, use digits instead of dashes

Reply:
Updated to digits